### PR TITLE
adding option to force reboot, ignoring active allerts

### PR DESCRIPTION
**Pull request by lunarfs**
we have a kubernetes cluster running on Azure, sometimes we see high values for softIRQ, we resolve this issue by rebooting the nodes, we want to performe this action even if active prometheus alerts exists. when we see this high values for softIRQ we touches a separate file on the host and the node should restart (despite active alerts exists in prometheus)

### CODE REVIEW

File: cmd/kured/main.go [hunk 1]
```diff
@@ -108,6 +110,18 @@ func sentinelExists() bool {
 		return false // unreachable; prevents compilation error
 	}
 }
+func forceRebootsentinelExists() bool {
```
Comment (bboreham) [inline cmd/kured/main.go]: please run `go fmt` on the code

Comment (lunarfs) [reply cmd/kured/main.go]: thanx for the feedback, i have now run go fmt on the code, and fixed a bug where forcereboot was only performed if reboot was also requiered

Comment (lunarfs) [reply cmd/kured/main.go]: @awh What is your take on this , how does this fit the overall roadmap? do you eventually have other suggestions?

**Comment by SerialVelocity** [comment]
@lunarfs It might be worth just supporting the word "force" inside the existing sentinel file? It looks like this covers issue #110 I just filed

**Comment by evrardjp** [comment]
@SerialVelocity that might indeed be simpler, as only the reboot blocked function would need changing by checking the content of the file.

**Comment by paprickar** [comment]
I would be also interested to have this merged.
@bboreham  / @awh do you think we can have that merged soon?

**Pull request closed without merging**